sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/GraphResourcesOperationsTests.cs

```diff
@@ -0,0 +1,162 @@
+                                new List<CompositePath>
+                                {
+                                    new CompositePath { Path = "/orderByPath3", Order = CompositePathSortOrder.Ascending },
+                                    new CompositePath { Path = "/orderByPath4", Order = CompositePathSortOrder.Descending }
+                                }
+                            }, new List<SpatialSpec> { new SpatialSpec ( "/*", new List<SpatialType> { new SpatialType("Point") } ) });
+
+            ContainerPartitionKey containerPartitionKey = new ContainerPartitionKey(new List<string> { "/address" }, "Hash", null);
+            IList<string> paths = new List<string>() { "/testpath" };
+            UniqueKey uk = new UniqueKey(paths);
+            IList<UniqueKey> uniqueKeys = new List<UniqueKey>();
+            uniqueKeys.Add(uk);
+            UniqueKeyPolicy uniqueKeyPolicy = new UniqueKeyPolicy(uniqueKeys);
+
+            ConflictResolutionPolicy conflictResolutionPolicy = new ConflictResolutionPolicy(new ConflictResolutionMode("LastWriterWins"), "/path", "");
+
+            GremlinGraphCreateUpdateParameters gremlinGraphCreateUpdateParameters = new GremlinGraphCreateUpdateParameters(new GremlinGraphResource(gremlinGraphName, indexingPolicy, containerPartitionKey, -1, uniqueKeyPolicy, conflictResolutionPolicy), new CreateUpdateOptions(sampleThroughput, default));
+
+            Response<GremlinGraphGetResults> gremlinResponse = await WaitForCompletionAsync(await CosmosDBManagementClient.GremlinResources.StartCreateUpdateGremlinGraphAsync(resourceGroupName, databaseAccountName, databaseName, gremlinGraphName, gremlinGraphCreateUpdateParameters));
+            GremlinGraphGetResults gremlinGraphGetResults = gremlinResponse.Value;
+            Assert.NotNull(gremlinGraphGetResults);
+
+            VerifyGremlinGraphCreation(gremlinGraphGetResults, gremlinGraphCreateUpdateParameters);
+
+            var gremlinPageableResultsTask = CosmosDBManagementClient.GremlinResources.ListGremlinGraphsAsync(resourceGroupName, databaseAccountName, databaseName).ToEnumerableAsync();
+            List<GremlinGraphGetResults> gremlinGraphs = gremlinPageableResultsTask.ConfigureAwait(false).GetAwaiter().GetResult();
+            Assert.NotNull(gremlinGraphs);
+        }
+
+        private void VerifyGremlinGraphCreation(GremlinGraphGetResults gremlinGraphGetResults, GremlinGraphCreateUpdateParameters gremlinGraphCreateUpdateParameters)
+        {
+            Assert.AreEqual(gremlinGraphGetResults.Resource.Id, gremlinGraphCreateUpdateParameters.Resource.Id);
+            Assert.AreEqual(gremlinGraphGetResults.Resource.IndexingPolicy.IndexingMode.Value.ToString().ToLower(), gremlinGraphCreateUpdateParameters.Resource.IndexingPolicy.IndexingMode.Value.ToString().ToLower());
+            //Assert.AreEqual(gremlinGraphGetResults.Resource.IndexingPolicy.ExcludedPaths, gremlinGraphCreateUpdateParameters.Resource.IndexingPolicy.ExcludedPaths);
```

Comment:
Do not have comment out code in production code.

Reply:
Removed